README.md

```diff
@@ -114,6 +90,8 @@ A related option is VS Code Dev Containers, which will open the project in your
     - [Powershell 7+ (pwsh)](https://github.com/powershell/powershell) - For Windows users only.
       - **Important**: Ensure you can run `pwsh.exe` from a PowerShell terminal. If this fails, you likely need to upgrade PowerShell.
 
+#### Installation
```

Comment:
The installation heading confuses me, since the prerequisites also need to installed. I liked doing the numbered list as it's clearer what steps need to be done for local dev. Otherwise devs get confused as to what's needed for each environment.

Reply:
Fixed